client/src/pages/email-sign-up.js

```diff
@@ -55,74 +53,43 @@ class AcceptPrivacyTerms extends Component {
 
   handleSubmit(e) {
     e.preventDefault();
-    const { privacyPolicy, termsOfService, quincyEmail } = this.state;
-    if (!privacyPolicy || !termsOfService) {
-      return null;
-    }
+    const { quincyEmail } = this.state;
+
     return this.props.acceptTerms(quincyEmail);
   }
 
+  componentWillUnmount() {
+    // if a user navigates away from here we should set acceptedPrivacyTerms
+    // to true (so they do not get pulled back) without changing their email
+    // preferences (hence the null payload)
+    // This ensures the user has to click the checkbox and then click the
+    // 'Continue...' button to sign up.
+    if (!this.props.acceptedPrivacyTerms) {
+      this.props.acceptTerms(null);
+    }
+  }
+
   render() {
     const { acceptedPrivacyTerms } = this.props;
     if (acceptedPrivacyTerms) {
-      return <RedirectHome />;
+      return <RedirectToLearn />;
     }
-    const { privacyPolicy, termsOfService, quincyEmail } = this.state;
+    const { quincyEmail } = this.state;
     return (
       <Fragment>
         <Helmet>
-          <title>Privacy Policy and Terms of Service | freeCodeCamp.org</title>
+          <title>Email Sign Up | freeCodeCamp.org</title>
         </Helmet>
         <Spacer size={2} />
         <Row className='text-center'>
           <Col sm={8} smOffset={2} xs={12}>
```

Comment:
These do not look right to me. Or, I might be missing something. Either way @ahmadabdolsaheb can you check the grid layout is the correct on this page before I test this.